### PR TITLE
fix eslint error (close #585)

### LINKED ISSUE #585: Fix eslint error 'Parsing error: 'import' and 'export' may appear only with 'sourceType: module''

### PR DESCRIPTION
**Pull request by miherlosev**
The problem that described in #585 was fixed by refreshing node modules.
But the new linting problem is appeared.
@LavrovArtem @AlexanderMoskovkin

### CODE REVIEW

File: src/client/sandbox/code-instrumentation/properties/index.js [hunk 1]
```diff
@@ -180,8 +180,14 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
...
                 get: el => LocationAccessorsInstrumentation.isLocationWrapper(el) ? el.href : PropertyAccessorsInstrumentation._getUrlAttr(el, 'href'),
-                set: (el, value) => LocationAccessorsInstrumentation.isLocationWrapper(el) ?
-                                    el.href = destLocation.resolveUrl(value, document) : el.setAttribute('href', value)
+                set: (el, value) => {
+                    if (LocationAccessorsInstrumentation.isLocationWrapper(el))
+                        el.href = destLocation.resolveUrl(value, document);
+                    else
+                        el.setAttribute('href', value);
+
+                    return void 0;
```
Comment (inikulin) [inline src/client/sandbox/code-instrumentation/properties/index.js]: It should return `href`

Comment (miherlosev) [inline src/client/sandbox/code-instrumentation/properties/index.js]: more precisely:
if `el.href=<value>` is called then we should return  <value>.
else retrun `void 0`.

tmp

Comment (inikulin) [inline src/client/sandbox/code-instrumentation/properties/index.js]: BTW FYI @churkin @miherlosev @LavrovArtem WHATWG just standartized location object: https://html.spec.whatwg.org/multipage/browsers.html#the-location-interface Might be useful

**Comment by miherlosev** [comment]
FPR

**Comment by LavrovArtem** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**